CrowdFundingApplication/Views/Profile/Index.cshtml

```diff
@@ -0,0 +1,146 @@
+﻿@model CrowdFundingApplication.Models.User
```

Comment:
You cannot use domain models for mapping. In all cases.

Reply:
Has been edited